static/app/views/discover/queryList.tsx

```diff
@@ -44,7 +44,6 @@ import {
 type Props = {
   api: Client;
   location: Location;
-  onQueryChange: () => void;
```

Comment:
this could be removed because `useApiQuery` tracks queries declaratively, and this was only used to trigger a re-fetch after pushing to the `browserHistory`, which now works automatically.